docs/modules/ROOT/partials/getting-started/con-registry-logging.adoc

```diff
@@ -0,0 +1,47 @@
+[id="registry-logging"]
+= {registry} Logging
+
+{registry} logging configuration can be modified while {registry} it's running. There is a REST endpoint that allows to configure the log level for specific loggers, allowing for finer grained logging configurations.
+
+[id="configuring-registry-logging"]
+== Configuring {registry} Logging during runtime
```

Comment:
Suggest sentence style heading and some minor copy edits:
== Configuring {registry} logging at runtime

This section explains how to view and set {registry} log levels at runtime using the admin REST API.